lm-agent-snap/README.md

```diff
@@ -0,0 +1,76 @@
+# lm-agent-snap
+The License Managar Agent bundled into a Snap.
+
+# Installation instructions
+
+For installing from the remote store, run:
+```bash
+sudo snap install license-manager-agent
+```
+
+## Basic Usage
+
+This snap requires a few configuration values to be set before it can be used. These values are:
```

Comment:
I think we should change the wording here since most of these settings are not required to be changed. It might be good to break up the settings into two groups. The first could be the required settings (`oidc-client-id` and `oidc-client-secret`) and the second list could be the optional settings. What do you think?

Reply:
that's a good idea, thank you

Reply:
done in [13c89e3](https://github.com/omnivector-solutions/license-manager/pull/389/commits/13c89e37e4fca4d4f8cd3de0090c4f227cc7a738)